tests/test_feature_downscale.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import pandas as pd
+import pyam
+
+
+@pytest.mark.parametrize("variable", (
+    ('Primary Energy'),
+    (['Primary Energy', 'Primary Energy|Coal']),
+))
+def test_downscale_region(aggregate_df, variable):
```

Comment:
just to make sure, this test is doing the following, correct?

1. downscale regional values of a global variable by population
2. pull out existing regional values
3. confirm they are equal

Does that mean that the population weights were specifically chosen to match the regional breakdown? Smart =)

Reply:
> Smart =)

Thank you (and yes, I played around with the test data to cover aggregation (one simple aggregation, one example with weights, one with region-level components) and downscaling both ways...